example/main.go

```diff
@@ -30,6 +36,57 @@ func init() {
 	flag.StringVar(externalMetadataFile, "m", "", "Description")
 }
 
+// swarmExtractor is an example metadata extractor for swarm that uses the service
```

Comment:
This should be in example/common.go so agent/datapath/standalone can be used to test encryption without launching full blown agent.